<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_account</class>
 <widget class="QDialog" name="create_account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QFrame#formFrame {
    background-color: #f5f5f5;
    border: 1px solid #ddd;
    border-radius: 8px;
    padding: 20px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>541</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    font-family: Arial, Helvetica, sans-serif;
    font-size: 12px;
    color: #444;
    font-weight: bold;

</string>
   </property>
   <property name="title">
    <string>Krijo llogari</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
font: 700 12pt &quot;Arial Rounded MT&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Emri:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>141</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 8px;
    border: 1px solid #bbb;
    border-radius: 4px;
    background-color: #fff;
    selection-background-color: #3498db;
    min-height: 25px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font: 700 12pt &quot;Arial Rounded MT&quot;;
color: rgb(255, 255, 255);
 font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>Id:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>141</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 8px;
    border: 1px solid #bbb;
    border-radius: 4px;
    background-color: #fff;
    selection-background-color: #3498db;
    min-height: 25px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial Rounded MT&quot;;
    
color: rgb(255, 255, 255);
    font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>141</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 8px;
    border: 1px solid #bbb;
    border-radius: 4px;
    background-color: #fff;
    selection-background-color: #3498db;
    min-height: 25px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 12pt &quot;Arial Rounded MT&quot;;
color: rgb(255, 255, 255);
    font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>Fjalekalim:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>141</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 8px;
    border: 1px solid #bbb;
    border-radius: 4px;
    background-color: #fff;
    selection-background-color: #3498db;
    min-height: 25px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>130</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 10px 15px;
    font-weight: bold;
    min-width: 100px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1a5276;
}
</string>
    </property>
    <property name="text">
     <string>Krijo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
